--- collector/processLink/convert/generic.js	2025-12-30 00:00:00.000000000 -0000
+++ collector/processLink/convert/generic-patched.js	2025-12-30 00:00:00.000000000 -0000
@@ -162,6 +162,12 @@
       launchOptions: {
         headless: launchConfig.headless,
         ignoreHTTPSErrors: true,
-        args: runtimeSettings.get("browserLaunchArgs"),
+        args: [
+          ...(runtimeSettings.get("browserLaunchArgs") || []),
+          "--no-sandbox",
+          "--disable-setuid-sandbox",
+        ],
+        executablePath:
+          process.env.PUPPETEER_EXECUTABLE_PATH || "/usr/bin/chromium-browser",
       },
       gotoOptions: {
